code1br/github-api
# Add morgan and winston packages to save logs

[06](https://trello.com/c/M3ZVCCux/6-log-de-requests)

 ### Description

This PR adds two packages, Morgan and Winston.
From now will be possible to save logs and also show them in the server log

 ### How to test
- Run the server and check if all requests have been shown on the terminal screen.
- Run the command `tail -f ./logs/all.log` and check all requests been saved in the file in real time

 ### Review

- [ ] Get approval from PM or stakeholders.
- [ ] Get approval from team QA
- [ ] If you add a data migration check consistency of the data.

 ### Pre-merge checklist

- [ ] Verify that the feature branch is up-to-date with `main` (if not - rebase it).

 ### Screenshots

| Before.                                      | After.                                       |
| -------------------------------------------- | -------------------------------------------- |
| x | ![Screenshot from 2022-10-21 11-28-29](https://user-images.githubusercontent.com/10809645/197222591-058721e4-8582-46df-ab04-e1db474b58e2.png) |
